packages/synapse-react-client/src/components/EntityLink.tsx

```diff
@@ -38,9 +38,13 @@ export const EntityLink = (props: EntityLinkProps) => {
     entityId = entityOrId
   }
 
-  const { data: fetchedEntity, error } = useGetEntity(entityId, versionNumber, {
-    enabled: !!entityId && typeof entityOrId === 'string',
-  })
+  const { data: fetchedEntity, error } = useGetEntityHeader(
+    entityId,
+    versionNumber,
+    {
+      enabled: !!entityId && typeof entityOrId === 'string',
+    },
+  )
```

Comment:
No need to fetch the entire entity. We can just fetch the header (which can also be pre-fetched in bulk for SynapseTable!)